src/js/views/globals/prelogin/prelogin.scss

```diff
@@ -85,13 +65,13 @@ body {
 }
 
 .prelogin__continue-button {
```

Comment:
This looks like it is reproducing our button classes.  Can we reuse the buttons module?

Reply:
Yup, good call. Just pushed an update that uses the buttons module.

Reply:
well I'd assume we could replace this class in the html with some `button-class`

Reply:
The closest button class is:

```css
.button--large {
  @extend .button;

  font-size: 16px;
  padding: 8px 40px;
}
```

But it doesn't have the correct `margin-top` or `padding`.

Reply:
ah I didn't realize `.button--green` was already in the html.. clearly wasn't doing anything before.

Reply:
Correct, before `'scss/modules/buttons.scss';` was imported, it wasn't doing anything.